apps/event-worker/src/resource-scan/aws/vpc.ts

```
@@ -0,0 +1,207 @@
+    },
+    metadata: omitNullUndefined({
+      [ReservedMetadataKey.ExternalId]: vpc.VpcId,
+      [ReservedMetadataKey.Links]: JSON.stringify({ "AWS Console": appUrl }),
+      "aws/region": region,
+      "aws/state": vpc.State,
+      "aws/is-default": vpc.IsDefault,
+      "aws/dhcp-options-id": vpc.DhcpOptionsId,
+      "aws/instance-tenancy": vpc.InstanceTenancy,
+      ...(vpc.Tags?.reduce(
+        (acc, tag) => ({
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)